src/com/lftechnology/java/training/alina/basic/packagejarexecute/NumberAdd.java

```diff
@@ -0,0 +1,35 @@
+
+package com.lftechnology.java.training.alina.basic.packagejarexecute;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * <p>NumberAdd Class is used to display the sum of two numbers</p> <p>Steps :
+ * </p> <ol type="a"> <li>Create a Java application AddNumber, which takes two
+ * number from User and prints sum.</li> <li>Create Executable Jar</li> <li>Run
+ * java application from command line which should take values from user.</li>
+ * <li>Print sum</li> <li>Create JavaDoc of Application</li> </ol>
+ * 
+ * @author Alina Shakya <alinashakya@lftechnology.com>
+ */
+public class NumberAdd {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)